### ISSUE TITLE
column d.adsrc does not exist pg 

### ISSUE THREAD

**Issue by osmanraifgunes** [sequelize/sequelize-auto#385]
Getting 

> column d.adsrc does not exist

error when generating modules.

My pg version is _PostgreSQL **12.0**, compiled by Visual C++ build 1914, 64-bit_ on windows.

I deleted line 115 (below code) under lib/dialect/index.js file and it works fine. 

`(SELECT d.adsrc AS extra FROM pg_catalog.pg_attribute a LEFT JOIN pg_catalog.pg_attrdef d ON (a.attrelid, a.attnum) = (d.adrelid,  d.adnum) \ WHERE NOT a.attisdropped AND a.attnum > 0 AND a.attrelid = o.conrelid \ LIMIT 1) \`

Is this the right solution?

Later edit : 

at documentation (https://www.postgresql.org/docs/8.0/catalog-pg-attrdef.html) it says :

> The adsrc field is historical, and is best not used, because it does not track outside changes that might affect the representation of the default value. Reverse-compiling the adbin field (with pg_get_expr for example) is a better way to display the default value.

So edited code to : 

`(SELECT pg_get_expr(d.adbin,m."oid") AS extra FROM pg_catalog.pg_attribute a LEFT JOIN pg_catalog.pg_attrdef d ON (a.attrelid, a.attnum) = (d.adrelid,  d.adnum) \ WHERE NOT a.attisdropped AND a.attnum > 0 AND a.attrelid = o.conrelid \ LIMIT 1) \`

I hope this is a better solution.
`

**Comment by Cauen** [comment]
This error appears on downgrading to pg 6, once pg 7+ is not working with this lib.
```
"pg": "6.1.5",
"pg-hstore": "^2.3.2",
"sequelize-auto": "^0.4.29"
```
This fix works very well to me :D

**Comment by steveschmitt** [comment]
Fixed in `sequelize-auto@0.5.0`
